translations/Translations.md

```diff
@@ -11,3 +11,4 @@
 | <img alt="Russian" title="Russian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ru.svg" width="22">                                                                                                                                                 | [Русский](./README.ru.md)                                               |
 <img alt="Italiano" title="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">                                                                                                                                                 | [Italiano](./README.it.md)                                               |
 | <img alt="Čeština" title="Čeština" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/cz.svg" width="22">                                                                                                                                               | [Čeština](README.cs.md)                                              |
+| <img alt="Български" title="Български" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/bg.svg" width="22">                                                                                                                                               | [Български](README.bg.md)                                              |
```

Comment:
Please, add the `./README.bg.md` instead just `README.bg.md`

Reply:
Fixing.

Btw, I copied the one above, so as you can see this means that `README.cs.md` is wrong as well.

Reply:
Yeah, I noticed it. I'll review it